matrix-sdk-android/src/main/java/fr/gouv/tchap/android/sdk/session/events/model/TchapEventType.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
The right copyright is New Vector Ltd here, because this is Tchap specification over Matrix

Reply:
Can you please check this point with the platform team before changing